packages/react-native/src/theme/createReactNativeTheme.ts

```diff
@@ -0,0 +1,55 @@
+// Internal Style Dictionary methods
+import deepExtend from 'style-dictionary/lib/utils/deepExtend';
+import get from 'lodash/get';
+import mapValues from 'lodash/mapValues';
+
+import { defaultTheme } from './defaultTheme';
+import { ColorMode, Theme, ReactNativeTheme } from './types';
+import { Tokens } from './tokens';
+
+const mapValuesDeep = (v: object | string, callback: Function): object =>
+  typeof v == 'object'
+    ? mapValues(v, (v) => mapValuesDeep(v, callback))
+    : (callback(v) as object);
+
+/**
+ * This will be used like `const myTheme = createReactNativeTheme({})`
+ * `myTheme` can then be passed to a Provider
+ * const myTheme = createReactNativeTheme({})
+ * const myOtherTheme = createReactNativeTheme({}, myTheme);
+ */
+export const createReactNativeTheme = (
+  theme?: Theme,
```

Comment:
Since this is the param for a customer to pass a custom theme, it should be `ReactNativeTheme` right?